### DISCUSSION TITLE
Change IP address inverter

### DISCUSSION THREAD

**Discussion by mvtslot** [robbinjanssen/home-assistant-omnik-inverter#131] [General]
Hi,

I had to change the IP address of my inverter, due to a move to another network. How do I change the IP address in the HASS configuration without loosing my data?

Thanks. 

Marco

**Comment by robbinjanssen** [comment]
Hi, unfortunately the only way to change the IP is to remove and re-add the integration. Im not sure what HA does when you keep the same entity names, maybe it keeps the history?
